pkg/catalogv2/git/download.go

```diff
@@ -84,14 +85,28 @@ func gitForRepo(secret *corev1.Secret, namespace, name, gitURL string, insecureS
 	if settings.InstallUUID.Get() != "" {
 		headers["X-Install-Uuid"] = settings.InstallUUID.Get()
 	}
+	// convert caBundle to PEM format because git requires correct line breaks, header and footer.
```

Comment:
Do we want to set `insecureSkipTLS` to false if `caBundle` is provided? I think the right thing is to do that if `caBundle` is provided as the expectation is that there is a fully valid certificate chain if provided, not that it still goes through if the user misconfigures this. We could also argue that this is guidance we can provide in the UI but still.

Reply:
good catch, it should be set to false